NEWS.md

```diff
@@ -2,16 +2,16 @@
 
 ## Major changes
 
-* @olivroy and @salim-b are now pkgdown authors in recognition of their contributions.
+* Added @olivroy and @salim-b as pkgdown authors in recognition of their contributions.
```

Comment:
User linking was not working when `@olivroy` was at the beginning of the line, likely a regex problem.